### ISSUE TITLE
1 frame per second on pi 3

### ISSUE THREAD

**Issue by bensuddenly** [gohai/processing-glvideo#52]
running processing 3, raspi3 , most recent update, and the pi camera v2.1
and the default simple capture script

any idea what could be causing this?
 and help greatly appreciated

**Comment by gohai** [comment]
Have you tested with the image from pi.processing.org?

**Comment by bensuddenly** [comment]
yep the raspistill stuff all works fine (480-1080 tested) and any normal image script runs fine
just noticed my void settings () are 720p but apart from that nothing is altered 

any ideas?

**Comment by gohai** [comment]
So it works with the provided image?

**Comment by bensuddenly** [comment]
yep everything works apart from running the camera in anything above 240p
